packages/zoe/test/unitTests/contracts/attestation/test-attestation.js

```diff
@@ -0,0 +1,278 @@
+// @ts-check
+
+/* global __dirname */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava';
+import { makeIssuerKit, AssetKind, AmountMath } from '@agoric/ertp';
+import { Far } from '@agoric/marshal';
+
+import bundleSource from '@agoric/bundle-source';
+import { E } from '@agoric/eventual-send';
+
+import { makeZoe } from '../../../../src/zoeService/zoe';
+import fakeVatAdmin from '../../../../tools/fakeVatAdmin';
+
+const attestationRoot = `${__dirname}/../../../../src/contracts/attestation/attestation`;
+
+test('attestation contract basic tests', async t => {
+  const bundle = await bundleSource(attestationRoot);
+
+  const zoe = makeZoe(fakeVatAdmin);
+  const installation = await E(zoe).install(bundle);
+
+  const bldIssuerKit = makeIssuerKit('BLD', AssetKind.NAT, {
+    decimalPlaces: 6,
+  });
+  const uBrand = bldIssuerKit.brand;
+  const uIssuer = bldIssuerKit.issuer;
+
+  const issuerKeywordRecord = harden({ Underlying: uIssuer });
+
+  let currentTime = 10n;
+
+  const mockCosmos = Far(
+    'cosmos',
+    /** @type {{getAccountState: (address: Address) => {total: Amount,
```

Comment:
Update type to accept a `Brand` denom argument.

Reply:
Will do